src/components/learning-units-section/complete-learning-unit-toogle/CompleteLearningUnitToogle.module.scss

```diff
@@ -4,3 +4,10 @@
   align-items: center;
   justify-content: space-around;
 }
+
+.button_without_style {
+  background: none !important;
```

Comment:
it's weird that you have to use !important.

Reply:
yah, it's weird, but there is no easy way to make it work without !important.